File: app/src/main/java/seucaioo/com/br/testeretrofit/retrofit/DataService.java
package seucaioo.com.br.testeretrofit.retrofit;

import retrofit2.Call;
import retrofit2.http.GET;
import seucaioo.com.br.testeretrofit.model.DatasResponse;

public interface DataService {
    @GET("data.json")
    Call<DatasResponse> getDatas();
}

File: app/src/main/java/seucaioo/com/br/testeretrofitkotlin/retrofit/DataService.kt
package seucaioo.com.br.testeretrofitkotlin.retrofit

import retrofit2.Call
import retrofit2.http.GET
import seucaioo.com.br.testeretrofitkotlin.model.DataResponse

interface DataService {

    @GET("data.json")
    fun getData() : Call<DataResponse>
}